tests/testsuite/freshness.rs

```diff
@@ -2381,8 +2385,21 @@ fn linking_interrupted() {
 
     // Interrupt the child.
     child.kill().unwrap();
-    // Note: rustc and the linker are still running, let them exit here.
-    conn.write(b"X").unwrap();
+    child.wait().unwrap();
+    // Note: rustc and the linker may still be running because we didn't kill
+    // the entire process group. Normally, when a user hits Ctrl-C, everything
+    // is killed. However, setting up process groups in a cross-platform test
+    // is a pain, and there's no easy way to know when everything has been
+    // killed. This write will tell them to exit, pretending that they died
+    // before finishing. Ignore the error, because (sometimes?) on Windows
+    // everything is already killed.
+    let _ = conn.write(b"X");
```

Comment:
I think this could just be `drop(conn)` to indicate to anyone blocked on it that the other side is gone?